README.md

```diff
@@ -42,7 +42,11 @@ Install `gotop`, `gotop-bin`, or `gotop-git` from the AUR.
 ```
 pkg install gotop
 ```
+### Ubuntu 18
```

Comment:
Also in my opinion `snap` works on every ubuntu version, so changing Ubuntu 18 to Ubuntu will be better.

Reply:
Also, isn't snap cross platform? So it should work on any distro with snap installed. If so, then we should change the title to `Snap`.